CMakeModules/AF_vcpkg_options.cmake

```diff
@@ -7,14 +7,27 @@
 
 set(ENV{VCPKG_FEATURE_FLAGS} "versions")
 set(ENV{VCPKG_KEEP_ENV_VARS} "MKLROOT")
+set(VCPKG_MANIFEST_NO_DEFAULT_FEATURES ON)
+
+set(VCPKG_OVERLAY_TRIPLETS ${ArrayFire_SOURCE_DIR}/CMakeModules)
```

Comment:
What is the intent of setting triplets with our cmake folder ?

Reply:
It allows us to make freetype as a static library and dynamic library for anything else.

Reply:
oh the triplet change for freetype. Forgot that file is over there.

Reply:
ahh shoot. I didn't update that variable. it should be CMakeModuels/vcpkg-triplets/. Will fix soon.